pkg/model/flow_content.go

```diff
@@ -84,12 +84,17 @@ func (p *BpfFlowContent) AccumulateAdditional(other *ebpf.BpfAdditionalMetrics)
 	if p.AdditionalMetrics.FlowRtt < other.FlowRtt {
 		p.AdditionalMetrics.FlowRtt = other.FlowRtt
 	}
+	// Network events
 	for _, md := range other.NetworkEvents {
 		if !AllZerosMetaData(md) && !networkEventsMDExist(p.AdditionalMetrics.NetworkEvents, md) {
 			copy(p.AdditionalMetrics.NetworkEvents[p.AdditionalMetrics.NetworkEventsIdx][:], md[:])
 			p.AdditionalMetrics.NetworkEventsIdx = (p.AdditionalMetrics.NetworkEventsIdx + 1) % MaxNetworkEvents
 		}
 	}
+	// Packet Translations
```

Comment:
it would be interesting to know if there's information potentially lost here. Eg. if two map/cpus report two different things for the same flow. idk if that's possible.

Reply:
also, do you know if zoneid can be 0 (other than when it's unset) ? If so, might be more efficient to check for existing zoneid, than checking for non-zero IPs

Reply:
yes zoneid of 0 is valid